wsgi/odyssy/odyssy/urls.py

```diff
@@ -18,5 +18,6 @@
 
 urlpatterns = [
     url(r'^admin/?', include(admin.site.urls)),
-    url(r'^', include('basic.urls'))
+    url(r'^events/', include('events.urls')),
```

Comment:
I have a doubt over this regex shouldnt it be `r'^events/?'`
same with the link below